[2022-01-29 12:36:26,219] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2020-06-02T06:00:00+00:00 [queued]>
[2022-01-29 12:36:26,272] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2020-06-02T06:00:00+00:00 [queued]>
[2022-01-29 12:36:26,272] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 12:36:26,272] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-01-29 12:36:26,272] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 12:36:26,323] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2020-06-02 06:00:00+00:00
[2022-01-29 12:36:26,350] {standard_task_runner.py:52} INFO - Started process 4688 to run task
[2022-01-29 12:36:26,390] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'local_to_gcs_task', 'scheduled__2020-06-02T06:00:00+00:00', '--job-id', '428', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmp8rp39zj0', '--error-file', '/tmp/tmpy0nv6td_']
[2022-01-29 12:36:26,401] {standard_task_runner.py:77} INFO - Job 428: Subtask local_to_gcs_task
[2022-01-29 12:36:26,660] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2020-06-02T06:00:00+00:00 [running]> on host 85e1117cfb12
[2022-01-29 12:36:26,912] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2020-06-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-06-02T06:00:00+00:00
[2022-01-29 12:36:28,895] {python.py:175} INFO - Done. Returned value was: None
[2022-01-29 12:36:28,990] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=local_to_gcs_task, execution_date=20200602T060000, start_date=20220129T123626, end_date=20220129T123628
[2022-01-29 12:36:29,227] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-29 12:36:29,420] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
